charts/newrelic-pixie/Chart.yaml

```diff
@@ -1,15 +1,18 @@
 apiVersion: v1
 description: A Helm chart for the New Relic Pixie integration.
 name: newrelic-pixie
-version: 1.3.2
-appVersion: 1.3.0
+version: 1.4.0
+appVersion: 1.4.0
 home: https://hub.docker.com/u/newrelic
 sources:
   - https://github.com/newrelic/
 engine: gotpl
 icon: https://newrelic.com/assets/newrelic/source/NewRelic-logo-square.svg
 maintainers:
-  - name: douglascamata
+  - name: paologallinaharbur
```

Comment:
The list changed https://github.com/newrelic/helm-charts/blob/master/charts/nri-bundle/Chart.yaml#L8

Reply:
😢

Reply:
Updated.